pkg/gardenlet/controller/factory.go

```diff
@@ -156,7 +165,7 @@ func (f *GardenletControllerFactory) Run(ctx context.Context) error {
 	var (
 		controllerInstallationController = controllerinstallationcontroller.NewController(f.clientMap, f.k8sGardenCoreInformers, f.cfg, f.recorder, gardenNamespace, f.gardenClusterIdentity)
 		seedController                   = seedcontroller.NewSeedController(f.clientMap, f.k8sGardenCoreInformers, f.k8sInformers, f.healthManager, secrets, imageVector, componentImageVectors, f.identity, f.cfg, f.recorder)
```

Comment:
Here we still use the statically fetched garden secrets, mainly for the monitoring configuration IISIC.
Are there plans to also dynamically fetch those or is this not needed?

Reply:
Nice catch! I also adapted this handling in 7572fc06111be0e13a9f9912692d69c35a73ccff and improved logging slightly when reading the Garden secrets.